imageio/core/fetching.py

```diff
@@ -31,7 +32,7 @@ class NeedDownloadError(IOError):
     """
 
 
-def get_remote_file(fname, directory=None, force_download=False, auto=True):
+def get_remote_file(fname, directory=None, force_download=False, auto=True, proxy_address=False):
```

Comment:
The argument is of string type, and is optional, so I think a default value of None makes more sense.

Reply:
@almarklein - You've convinced me. It'll change the logic in fetching to use None, but it's a trivial change.